src/System.Management.Automation/namespaces/NavigationProviderBase.cs

```diff
@@ -375,44 +375,13 @@ protected string MakePath(string parent, string child, bool childIsLeaf)
 
```

Comment:
@iSazonov, your last commit had 130 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Extended FileSystem Item/Content Cmdlet Provider Tests.Valdiate New-Item parameters.Verify Directory + Whatif
```powershell
Expected an exception, with FullyQualifiedErrorId 'PathNotFound,Microsoft.PowerShell.Commands.GetItemCommand' to be thrown, but the FullyQualifiedErrorId was 'System.Management.Automation.PSArgumentException,Microsoft.PowerShell.Commands.GetItemCommand'. from /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1:900 char:15
    +             { Get-Item -Path $testDir -ErrorAction Stop } |
    +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 901
901:                 Should -Throw -ErrorId "PathNotFound,Microsoft.PowerShell.Commands.GetItemCommand"
```
Extended FileSystem Item/Content Cmdlet Provider Tests.Valdiate New-Item parameters.Verify File + Value
```powershell
Expected exactly 'testFile.txt', but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 918
918:             $result.Name | Should -BeExactly $testFile
```
Extended FileSystem Item/Content Cmdlet Provider Tests.Valdiate Get-Item parameters.Verify Force
```powershell
Expected exactly 'testFile.txt', but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 926
926:             $result.Name | Should -BeExactly $testFile
```
Extended FileSystem Item/Content Cmdlet Provider Tests.Valdiate Get-Item parameters.Verify Path Wildcard
```powershell
Expected exactly 'testFile2.txt', but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 931
931:             $result.Name | Should -BeExactly $testFile2
```
Extended FileSystem Item/Content Cmdlet Provider Tests.Valdiate Get-Item parameters.Verify Include
```powershell
Expected exactly 'testFile2.txt', but got $null.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 936
936:             $result.Name | Should -BeExactly $testFile2
```